src/components/ui/BlogItem.jsx

```diff
@@ -1,11 +1,87 @@
-const Blog = ({ data: blog }) => {
+import { format } from 'date-fns';
+import lt from 'date-fns/locale/lt';
+import Image from 'next/image';
+import Link from 'next/link';
+import { shuffleArray } from './ShuffleArray';
+
+const BlogItem = ({ blog, styleLatest, index }) => {
+  const latestStyle = styleLatest && index !== 0;
+  const firstFromRecent = styleLatest && index === 0;
+  const categoryColors = [
```

Comment:
maybe adding color to category scheme in hygraph would make this a lot simpler:

https://tailwindcss.com/docs/customizing-colors

![image](https://github.com/MagicJourney-Labs/magicjourney-labs-app/assets/52066125/50a12d36-7cd5-4e03-915b-a75573956d31)

Reply:
I fixed it